src/main/resources/templates/overview.html

```diff
@@ -102,10 +102,9 @@
         <!-- Button to the directory above -->
         <div th:if="${directory.parentId} != null">
             <div class="material1-button-above-wrapper">
-                <form action="#" th:action="@{/material1/dir/{dirId}(dirId = ${directory.parentId})}"
-                      th:method="get">
-                    <input class="btn btn-outline-primary" type="submit" value="Ebene hoch">
-                </form>
+                <a class="btn btn-outline-primary" th:href="@{/material1/dir/{dirId}(dirId = ${directory.parentId})}">
+                    Ebene hoch
```

Comment:
Maybe we could use this one here https://icons.getbootstrap.com/icons/arrow-90deg-up/